docs/ref/models/querysets.txt

```diff
@@ -1705,9 +1705,11 @@ them::
     >>> Person.objects.select_related('hometown').select_for_update().exclude(hometown=None)
     <QuerySet [<Person: ...)>, ...]>
 
-Currently, the ``postgresql``, ``oracle``, and ``mysql`` database
-backends support ``select_for_update()``. However, MySQL doesn't support the
-``nowait``, ``skip_locked``, and ``of`` arguments.
+Currently, the ``PostgreSQL``, ``Oracle``, ``MySQL``, and ``MariaDB`` database
```

Comment:
We mention database backends, not databases. MySQL and MariaDB share the same database backend, so this change is not necessary.